UndertaleModLib/Util/FileBinaryReader.cs

```diff
@@ -199,6 +199,7 @@ public string ReadGMString()
                 throw new IOException("Reading out of bounds");
 #endif
             string res;
+            // TODO: shouldn't these accomplish the same thing?
```

Comment:
I think this is written like this because the stack has limited memory, and thus is limited to only storing 1024 bytes.

Reply:
Do arrays of primitive types not land up on the stack?

Reply:
nvm they dont. will remove that then